packages/changed-elements-react/src/widgets/comparisonJobWidget/components/VersionCompareDialogProvider.tsx

```diff
@@ -0,0 +1,117 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React from "react";
+import { JobAndNamedVersions } from "../models/ComparisonJobModels";
+
+/** Comparison Job Update Type
+*  - "JobComplete" = job is completed
+*  - "JobError" = job error
+*  - "JobProgressing" = job is started
```

Comment:
The doc doesn't match what's in the type below. If this is called multiple times, `JobProcessing` is fine, but if it's only called once at start, I would call it `JobStarted`

```suggestion
*  - "JobProcessing" = job is started
```